docker-compose.yml

```diff
@@ -22,7 +22,7 @@ x-cm-variables: &cm
     PALACE_CELERY_CLOUDWATCH_STATISTICS_DRYRUN: "true"
 
     # Set up the environment variables used for testing as well
-    SIMPLIFIED_TEST_DATABASE: "postgresql://palace:test@pg:5432/circ"
+    PALACE_TEST_DATABASE_URL: "postgresql://palace:test@pg:5432/circ"
```

Comment:
Minor: Do we want to omit the database name part from this URL, since we're running tests in parallel?

Reply:
This does work generally, but we can't do it in this case. We connect to whatever DB is passed into the env var, to create the worker DBs. If no DB is passed in, PG defaults to connecting to a DB with the same name as the users. But in this case when we create the container, we tell PG to name the users DB something different, `circ` instead. So if we don't pass in `circ` here, the calls will fail with an error something like `Unable to connect DB palace does not exist.`